rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/ast/JavaTypeReferenceModel.java

```diff
@@ -18,8 +18,21 @@
 
     public static final String TYPE = "JavaTypeReference";
     public static final String REFERENCE_TYPE = "referenceType";
+    public static final String RESOLVED_SOURCE_SNIPPIT = "resolvedSourceSnippit";
 
     /**
+     * Gets the snippit referenced by this {@link FileLocationModel}.
+     */
+    @Property(RESOLVED_SOURCE_SNIPPIT)
+    void setResolvedSourceSnippit(String source);
```

Comment:
What do you mean exactly? I would say it is easier to understand that SOURCE_SNIPPIT is a raw part of the source and it would also correlate with the xml layer we have. BTW thanks for taking time looking at this PR